### ISSUE TITLE
Fancy Games

### ISSUE THREAD

**Issue by Creeperkek** [bebeli555/CookieClient#2]
I'm not sure if I'm being a boomer or what, but when either of the games (snake and/or tetris) are opened, there isn't an apparent way to close them. When the gui is temporarily closed then reopened, they remain and if clicked on, it only restarts the games.

: P sorry for inconvenience, the games would probably get annoying unless there actually is a way to close them, I just can't figure out how to close them.

**Comment by bebeli555** [comment]
You just need to toggle off the module the same way you enabled it. I can update it so the games get closed when u close the gui tho

**Comment by Creeperkek** [comment]
That would be awesome : )
